var/cache/.gitkeep

```diff
@@ -1 +0,0 @@
-# Keep cache folder
```

Comment:
why remove this text/content ?

Reply:
hum must have been deleted when rebasing...